hedera-state-validator/src/main/java/com/hedera/statevalidation/util/StateUtils.java

```diff
@@ -0,0 +1,312 @@
+            @NonNull final PlatformContext platformContext,
+            @NonNull final ServicesRegistry servicesRegistry) {
+        final Configuration configuration = platformContext.getConfiguration();
+        final ServiceMigrator serviceMigrator = new OrderedServiceMigrator();
+        final PlatformStateFacade platformFacade = PlatformStateFacade.DEFAULT_PLATFORM_STATE_FACADE;
+        final SemanticVersion version = platformFacade.creationSoftwareVersionOf(state);
+
+        PlatformStateService.PLATFORM_STATE_SERVICE.setAppVersionFn(v -> version);
+
+        // previousVersion and currentVersion are the same!
+        serviceMigrator.doMigrations(
+                (MerkleNodeState) state,
+                servicesRegistry,
+                version,
+                version,
+                configuration,
+                configuration,
+                new FakeStartupNetworks(Network.newBuilder().build()),
+                new StoreMetricsServiceImpl(new NoOpMetrics()),
+                new ConfigProviderImpl(),
+                platformFacade);
+    }
+
+    // Used for lambda shorthands
+    private static State getState() {
+        return deserializedSignedState.reservedSignedState().get().getState();
+    }
+
+    // Uses cached JSON codecs
+    @SuppressWarnings("unchecked")
+    public static String keyToJson(OneOf<StateKey.KeyOneOfType> key) {
+        return lookupKeyCodecFor(key).toJSON(key.value());
+    }
+
+    // Uses cached JSON codecs
+    @SuppressWarnings("unchecked")
+    public static String valueToJson(OneOf<StateValue.ValueOneOfType> value) {
+        return lookupValueCodecFor(value).toJSON(value.value());
+    }
+
+    // Uses cached JSON codecs
+    public static JsonCodec lookupKeyCodecFor(OneOf<StateKey.KeyOneOfType> key) {
+        return keyCodecsById.computeIfAbsent(key.kind().protoOrdinal(), id -> StateUtils.getCodecFor(key.value()));
+    }
+
+    // Uses cached JSON codecs
+    public static JsonCodec lookupValueCodecFor(OneOf<StateValue.ValueOneOfType> value) {
+        return valueCodecsById.computeIfAbsent(
+                value.kind().protoOrdinal(), id -> StateUtils.getCodecFor(value.value()));
+    }
+
+    // Extract JSON codec from a pbj object
+    public static JsonCodec getCodecFor(@NonNull final Object pbjObject) {
+        try {
+            final Field jsonCodecField = pbjObject.getClass().getDeclaredField("JSON");
+            return (JsonCodec) jsonCodecField.get(null);
+        } catch (Exception e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    public static int stateIdFor(@NonNull final String serviceName, @NonNull final String stateKey) {
+        Objects.requireNonNull(serviceName);
+        Objects.requireNonNull(stateKey);
+
+        int stateId = singletonStateIdFor(serviceName, stateKey);
+
+        if (stateId > 0) {
+            return stateId;
+        }
+
+        return stateIdFromStateKey(serviceName, stateKey);
```

Comment:
@imalygin I think we should throw exception here. Because now there is a catch: on `main` branch there is an example in README:
```
java -jar -DmaxObjPerFile=100000 ./validator-0.65.0.jar /path/to/round export /path/to/result AccountService ACCOUNTS
```

The problem is that there is no `AccountService` (I fixed it in this PR), and simple export command works just fine with that and creates file with data (where it exports whole state). On the other hand sorted export command does not create file at all. So, with not existing service, the behavior of two similar commands is completely different 🙂 
That's why I think it would be good to just fail if there is no such pair of service name and state key -- basically IllegalArgumentException.